sql/moz-fx-data-shared-prod/monitoring_derived/jobs_by_organization_v1/query.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+"""
+
+Put data from the projects below into a table.
+
+  mozdata.region-us.INFORMATION_SCHEMA.JOBS_BY_ORGANIZATION,
```

Comment:
See previous comment about simplifying this since JOBS_BY_ORGANIZATION is the same no matter which project is used to query it.